data-es7.js

```diff
@@ -320,8 +320,10 @@ exports.tests = [
   link: 'https://gist.github.com/WebReflection/9353781',
   category: 'strawman',
   exec: function () {/*
-    var B = Symbol('b');
-    var O = Object.defineProperty({a: 1, [B]: 2}, 'c', {value: 3});
+    var object = {a: 1};
+    var B = typeof Symbol === 'function' ? Symbol('b') : 'b';
+    object[B] = 2;
+    var O = Object.defineProperty(object, 'c', {value: 3});
```

Comment:
That's what the test was already doing. If you think it makes a difference, I could definitely remove `var O = `, but I don't think it matters.